webapp-django/crashstats/authentication/management/commands/makesuperuser.py

```diff
@@ -9,11 +9,11 @@
 
 from django.contrib.auth.models import Group, User
 from django.core.management.base import BaseCommand, CommandError
-from django.utils.six.moves import input
 
 
 def get_input(text):
-    return input(text).strip()
+    """Get user input, or mock it for tests."""
+    return input(text)
```

Comment:
We dropped the `.strip()` here. Having the `.strip()` handles copy-and-paste issues where the user inadvertently adds whitespace.

Reply:
Ah you caught that 😁 . I thought it was weird we were double-striping the output:

https://github.com/mozilla-services/socorro/blob/b684523243ca23b9e85578e3cea855ff775d7cbf/webapp-django/crashstats/authentication/management/commands/makesuperuser.py#L31

Reply:
Oh weird. I thought I would have done the stripping in `get_input()` and not at the usage point. Ok--past me probably knew best maybe or something.

Reply:
My guess is that tests were failing with a mocked ``get_input()``

Reply:
Or maybe pipe a list of emails, with possible whitespace, although I'm not sure it would work, and why would you do that?

Reply:
Adding it back, this PR should not introduce code changes.